custom/aaa/views.py

```diff
@@ -342,90 +366,112 @@ def post(self, request, *args, **kwargs):
         data = {}
 
         if sub_section == 'person_details':
-            person = dict(
-                name='Reena Kumar',
-                gender='Female',
-                status='Pregnant Woman',
-                dob=date(1991, 5, 11),
-                marriedAt=25,
-                aadhaarNo='Yes'
-            )
-            other = dict(
-                address='J-142, Saket, New Delhi, Delhi',
-                subcentre='Rasidpur',
-                village='Rasidpur',
-                anganwadiCentre='Aspataal Ward',
-                phone='844-860-4774',
-                religion='Hinduk',
-                caste='Sudra',
-                bplOrApl='BPL',
+            person_model = Woman if section != 'child' else Child
+
+            extra_select = {
+                'gender': 'sex',
+                'aadhaarNo': 'has_aadhar_number',
+                'address': 'hh_address',
+                'phone': 'contact_phone_number',
+                'religion': 'hh_religion',
+                'caste': 'hh_caste',
+                'bplOrApl': 'hh_bpl_apl',
+                'subcentre': 'sc_id',
+                'village': 'village_id',
+                'anganwadiCentre': 'awc_id'
+            }
```

Comment:
I think this section could be expressed more simply:

```python
values = ['sex', 'has_aadhar_number', 'hh_address', 'contact_phone_number', ...]
if section == 'child':
   values += ['extra_columns]
else:
   values += ['extra_columns']

person = person_model.objects.values(*values).get(domain=domain, person_case_id=beneficiary_id)
# update person with location names
return person
```

This will also require changing the javascript code to accept these column names instead of the ones currently being used. This would follow our general convention in HQ of passing the python variable names to javascript without transforming them to camelCase